bin/isolated-functions.sh

```diff
@@ -380,9 +380,6 @@ eend() {
 	if [[ -v EBEGIN_EEND ]] ; then
 		local caller="${FUNCNAME[1]}"
 		local tos="${EBEGIN_EEND[-1]}"
```

Comment:
These could also just be dropped then.

You could also consider changing the EBEGIN_EEND stack to a simple counter and increment in ebegin, then decrement here in eend.

Reply:
ebegin also uses the stack to see if there's an unclosed call to ebegin from another function. If so, a warning is emitted that it's missing a call to eend.